docs/specviz2d/plugins.rst

```diff
@@ -46,6 +46,11 @@ Spectral Extraction
 The Spectral Extraction plugin exposes `specreduce <https://specreduce.readthedocs.io>`_
 methods for tracing, background subtraction, and spectral extraction from 2D spectra.
 
+To interact with the plugin via the API in a notebook, access the plugin object via::
+
+  sp_ext = viz.app.get_tray_item_from_name('spectral-extraction')
```

Comment:
What do you think of adding a `viz.spectral_extraction` or `viz.spectral_extraction_plugin` attribute to the Specviz2D helper that is set to this, so that the user doesn't have to go through the app-level incantation?

Reply:
How would you feel about re-considering #1401?  It's _slightly_ more verbose than what you suggest, but I think more friendly than the current app-level method.

Reply:
Hmm....I'm fine with thinking about this harder in the context of 1401 and not holding up this PR.

Reply:
I agree that documenting any methods on `viz.app` is a bit of a slippery slope, but it is one we've already started sliding down...